README.rst

```diff
@@ -188,6 +188,8 @@ second usage. Save the result to a list if the result is needed multiple times.
 
 **B033**: Sets should not contain duplicate items. Duplicate items will be replaced with a single item at runtime.
 
+**B034**: Calls to `re.sub`, `re.subn` or `re.split` should pass `flags` or `output`/`maxsplit` as keyword arguments. Not doing so is a very common source of confusion.
```

Comment:
I had to go read the issue to understand what the confusion was here. Let’s try make this clearer to save people hunting the issue down.
```suggestion
**B034**: Calls to `re.sub`, `re.subn` or `re.split` should pass `flags` or `output`/`maxsplit` as keyword arguments. Not doing so is a very common source of confusion due to the positions of these arguments differing in the `re` module APIs.
```
- Happy for better wording.

Reply:
Isn't it just as much, if not more, about people not being aware that `count`/`maxsplit` are arguments at all, and just being used to passing `flags` as the third argument to all the other functions in `re`?
I agree with making it more clear though